### PR TITLE
1055 add link to genspectrum

### LINKED ISSUE #1055: Add link to GenSpectrum
TODO

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #1055

### Summary

Adds a banner to the "home" page and a navigation entry, which links to genspectrum.

Also includes refactoring of the header, to make it more readable.

### Screenshot

On the "home" page:
grafik

In the Drawer navigation:
grafik

### CODE REVIEW

**Comment by chaoran-chen** [comment]
Thanks Jonas, this looks great!

File: src/layout/base/TopNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { MdOutlineOpenInNew } from 'react-icons/md';
+import { GitHubButton, MastodonButton, TwitterButton } from './SocialIcons';
+
+export function TopNavigation(props: { hideInternalLinks: boolean | undefined }) {
+  return (
+    <nav id='right-nav-buttons' className='items-center justify-center '>
+      <ul className='ml-1 flex items-center gap-6 xl:gap-8'>
+        <li>
+          <NavigationLink path={'https://genspectrum.org'}>
+            <span className='inline-flex gap-1'>
```
Comment (fengelniederhammer) [inline src/layout/base/TopNavigation.tsx]:
```suggestion
            <span className='inline-flex items-center gap-1'>
```
Before:
grafik


After:
grafik

File: src/pages/ExplorePage.tsx [hunk 2]
```diff
@@ -96,3 +99,21 @@ export const ExplorePage = ({ isSmallScreen }: Props) => {
...
+
+function GenspectrumBanner() {
+  return (
+    <div className='w-full bg-blue-50 shadow-lg mt-4 rounded-xl p-4 dark:bg-gray-800 mx-2 mr-4'>
+      <h2>Investigate other pathogens</h2>
+      <p>
+        Explore <span className='font-bold'>Influenza A/H5N1, West Nile, RSV-A and RSV-B</span> on our new
+        interactive platform{' '}
+        <Link to={'https://genspectrum.org/'} className={'text-active-secondary'}>
+          <span className='inline-flex gap-1'>
```
Comment (fengelniederhammer) [inline src/pages/ExplorePage.tsx]:
```suggestion
          <span className='inline-flex items-center gap-1'>
```

**Review by fengelniederhammer — APPROVED** [review]
Thanks for the refactoring :)

**Pull request merged**